server/commands/expand.go

```diff
@@ -43,21 +40,44 @@ func (query *ExpandQuery) Execute(ctx context.Context, req *openfgapb.ExpandRequ
 
 	tk := tupleUtils.NewTupleKey(object, relation, "")
 
-	userset, err := query.getUserset(ctx, store, modelID, tk)
+	model, err := query.datastore.ReadAuthorizationModel(ctx, store, modelID)
 	if err != nil {
-		return nil, err
+		if errors.Is(err, storage.ErrNotFound) {
+			return nil, serverErrors.AuthorizationModelNotFound(modelID)
+		}
+
+		return nil, serverErrors.HandleError("", err)
 	}
 
-	model, err := query.datastore.ReadAuthorizationModel(ctx, req.GetStoreId(), req.GetAuthorizationModelId())
+	err = validation.ValidateObject(model, tk)
```

Comment:
Unfortunately this validation and the one below cannot be delegated to `validation.ValidateTuple` because we don't valid the `user` field here since we're expanding an `object#relation`. So this is about as consolidated as it gets for Expand.